src/tests/functional/shared_test_classes/include/shared_test_classes/base/layer_test_utils.hpp

```diff
@@ -11,6 +11,13 @@
 #include <tuple>
 #include <gtest/gtest.h>
 #include <ngraph/node.hpp>
+#include <ngraph/opsets/opset3.hpp>
+#include <ngraph/opsets/opset4.hpp>
+#include <ngraph/opsets/opset5.hpp>
+#include <ngraph/opsets/opset6.hpp>
+#include <ngraph/opsets/opset7.hpp>
+#include <ngraph/opsets/opset8.hpp>
+#include <ngraph/opsets/opset9.hpp>
```

Comment:
do we plan to eliminate all of this? 
having global headers in base header files for tests is overkill

Reply:
It should be done in the process of migration shared layer tests